doc/tutorials/menu/menu_05.py

```diff
@@ -155,7 +155,7 @@ def on_draw(self):
 class SubMenu(arcade.gui.UIMouseFilterMixin, arcade.gui.UIAnchorLayout):
     """Acts like a fake view/window."""
 
-    def __init__(self, title: str, input_text: str, toggle_label: str, dropdown_options: list[str], slider_label: str):
+    def __init__(self, title: str, input_text: str, toggle_label: str, dropdown_options: list, slider_label: str):
```

Comment:
Should this be `List[str]` with `from typing import List`?

Reply:
That's not legal in Python 3.8

Reply:
Updated